aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -157,6 +157,7 @@ module aptos_framework::object {
     }
 
     /// Emitted whenever the object's owner field is changed.
+    #[event]
```

Comment:
note: not sure reusing this struct or create a new one. seems unnecessary to me but a different opinion is welcomed.

Reply:
go for it...